azure-kusto-ingest/tests/sample.py

```diff
@@ -1,21 +1,55 @@
-"""Sample how to use Kusto Ingest client"""
+"""Samples on how to use Kusto Ingest client. Just Replace variables and run!"""
 
 from azure.kusto.data.request import KustoConnectionStringBuilder
 from azure.kusto.ingest import KustoIngestClient, IngestionProperties, FileDescriptor, BlobDescriptor, DataFormat
 
-INGESTION_PROPERTIES = IngestionProperties(database="database name", table="table name", dataFormat=DataFormat.csv)
+ingestion_props = IngestionProperties(database="{database_name}", table="{table_name}", dataFormat=DataFormat.csv)
+client = KustoIngestClient("https://ingest-{cluster_name}.kusto.windows.net")
 
-INGEST_CLIENT = KustoIngestClient("https://ingest-<clustername>.kusto.windows.net")
+# there are more posbilities for authenticating
+# KCSB = KustoConnectionStringBuilder.with_aad_application_key_authentication(
```

Comment:
Maybe show here the use of authority ID?

Reply:
this is just an older branch, once I will rebase I will make sure that there would be such an example